src/components/WelcomeMessage.tsx

```diff
@@ -0,0 +1,8 @@
+const WelcomeMessage = (): JSX.Element => {
+    return (
+        <span>Hello, Name!</span>
+    )
```

Comment:
Can you also run the linter and the prettier formatter?

The commands are `npm run lint` and `npm run format`